css/index.css

```diff
@@ -1 +1,383 @@
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+    font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+.container {
+    max-width: 880px;
+    width: 100%;
+    /* TB LR */
+    margin: 0 auto;
+    overflow: hidden;
+}
+
+/* header */
+
+header {        
+    display: flex;
+    width: 100%;
+    vertical-align: top;
+    margin-top: 3%;
+    flex-direction: row;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+header nav {
+    width: 50%;
+    display: flex;
+    justify-content: space-evenly
+}
+header nav a {
+    font-size: 1.5rem;
+    color: gray;
+    text-decoration: none;
+}
+
+header nav a:hover {
+    text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+
+
+/* cta */
+.cta {
+    display:flex;
+    flex-wrap: nowrap;
+    align-items: center;
+    width: 100%;
+}
+
+.cta .cta-text {
+	width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    font-size: 77px;
+    margin-top: 10%;
+    text-align: center;
+    padding: 0px 10%;
+}
+
+.cta .cta-text button {
+	display: flex;
+	align-self: center;
+    background: white;
+    margin-top: 4%;
+    padding: 2% 8%;
+    font-size: 17px;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+.cta img {
+    margin-top: 7%;
+}
+
+.main-content {
+	display: flex;
+	flex-direction: column;
+    border-top: 1px solid black;
+    margin-top: 9%;
+    padding-top: 5%;
+}
+
+.main-content .top-content {
+  padding: 0 1%;
+  display: flex;
+}
+
+.main-content .top-content .text-content {
+  width: 50%;
+  margin-right: 4%;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 5%;
```

Comment:
Left and Right margins definitely need to be changed to percents, but not necessarily for the vertical margins.